src/test/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusinessIntegrationSpec.scala

```diff
@@ -18,48 +16,43 @@ class AgoraBusinessIntegrationSpec extends FlatSpec with BeforeAndAfterAll with
   }
 
   "Agora" should "be able to store a task configuration with a valid (extant) official docker image" in {
-    val entity = agoraBusiness.insert(testAgoraEntityWithValidOfficialDockerImageInWdl, mockAuthenticatedOwner.get).addManagers(Seq(mockAuthenticatedOwner.get))
-    val actual = agoraBusiness.findSingle(entity, Seq(entity.entityType.get), mockAuthenticatedOwner.get)
-    assert(actual === entity)
+    val entity = patiently(agoraBusiness.insert(testAgoraEntityWithValidOfficialDockerImageInWdl, mockAuthenticatedOwner.get)).addManagers(Seq(mockAuthenticatedOwner.get))
```

Comment:
`patiently` is a synonym for `Await.result`, and is necessary here since the calls to `agoraBusiness` and such used to wait internally but now return futures